src/main/java/org/kjy5/spork/Merger.java

```diff
@@ -129,10 +135,12 @@ private static void removeSoftContentInconsistencies(
    * @return The set of inconsistent PCSs.
    */
   private static Set<Pcs> getAllInconsistentPcs(Pcs pcs, ChangeSet changeSet) {
+    // Why is this a set?  Can duplicates exist?
```

Comment:
I should not be encountering duplicates, but I am using a set anyway since I do not need ordering.

Reply:
A set affirmatively enforces some properties.  If you want to communicate that ordering does not matter, then using a type like `Collection` is probably better.

Reply:
The abstract type `Collection` is what I meant.  Whatever the concrete (run-time) type, using `Collection` as the compile-time, declared type indicates that ordering and duplicates are not important.